test/test_HMMread.h

```diff
@@ -0,0 +1,156 @@
+/*
```

Comment:
What is this file for? Didn't you implement that already in readFile.cpp?

Reply:
ja merkwürdig eigentlich war da was drin..

Reply:
habe das file verwechselt. Die Moden werden in Attract noch einmal verändert (auf den gesamtbetrag der Moden normiert) dieser Test ist dafür gedacht diesen Teil der eigentlichen readHMM funktion zu überprüfen.